### DISCUSSION TITLE
visibility of AVAudioFormat toPCMBuffer function

### DISCUSSION THREAD

**Discussion by nightfall708** [kingslay/KSPlayer#252] [General]
Hi,

(I hope this is not the wrong place to ask this question)
Is there a reason toPCMBuffer private? Is it considered risky removing the `private` keyword and accessing it internally from AudioEnginePlayer?

**Comment by kingslay** [comment]
There is no risk. But the method toPCMBuffer has not been used. I don't know if it can actually be used. You can try and modify this method at will

**Reply by nightfall708** [reply]
Why do you think it might be not used? because of different formats?

**Reply by bannawat555** [reply]
C

**Reply by kingslay** [reply]
这个方法可以使用了。我有在一个地方使用了这个这个方法了。

**Comment by kingslay** [comment]
Because there is currently nowhere to call this function
